src/MessageDialog.Uno/MessageDialog.Uno.csproj

```diff
@@ -49,5 +50,5 @@
 		</None>
 	</ItemGroup>
 
-	<Import Project="..\..\Shared\Shared.projitems" Label="Shared" />
+	<Import Project="..\Shared\Shared.projitems" Label="Shared" />
```

Comment:
Shouldn't it be `MessageDialog.Shared` instead of `Shared`?

Reply:
For both the folder name and the label for the reference

Reply:
yep!

Reply:
Fixed!

Reply:
You forgot the label